test/Dialect/Tpp/tpp-bufferization.mlir

```diff
@@ -341,3 +341,54 @@ func.func @test_mlp_bf16_3_layer_1024(%arg0: tensor<256x1024xbf16>,
   %2 = tpp.relu (%1: tensor<256x1024xbf16>) -> tensor<256x1024xbf16>
   return %2 : tensor<256x1024xbf16>
 }
+
+// -----
+
+// CHECK: test_zero
+// CHECK-SAME: %[[ARG0:.+]]: memref<5x5xf32>
+func.func @test_zero(%arg0: tensor<5x5xf32>) -> tensor<5x5xf32> {
```

Comment:
Can you add either `CHECK-NOT: alloc` or `CHECK: alloc` to make sure we keep the behaviour over future changes?

Reply:
in `test_zero`? If we emit an alloc the CHECK will still fail.

Reply:
done in: https://github.com/plaidml/tpp-mlir/pull/485/commits/1531fd621045320d5d625a5095fff26836441d9a